### DISCUSSION TITLE
Parent Link displayed as Document Library Name

### DISCUSSION THREAD

**Discussion by clmmlc** [microsoft-search/pnp-modern-search#3832] [Q&A]
Hi There

I am currently trying to get a link to a doucmnets location to display as the libraries name in he search results

<a href="{{ParentLink}}" target="_blank">{{ParentLink}}</a> works great to return the parent location link 

But I would like the link to display as the document library name. Not display as the full URL.

![2024-07-01_11-20-42](https://github.com/microsoft-search/pnp-modern-search/assets/107613292/2474de35-d0be-46f7-9455-30e19ac983a1)

![2024-07-01_11-21-44](https://github.com/microsoft-search/pnp-modern-search/assets/107613292/04b25b73-791c-4b49-9483-5229cfcf2f18)

Does anyone have a way I could achieve this? Any help would be much appreciated!

**Comment by kasperbolarsen** [comment]
As far as I know the name of the library is not available in the dataset. I guess your best option will be to use Handlebars and split the ParentLink on / , this should work unless you have subsites ;-)
